src/main/java/com/fryrank/model/GetAllReviewsOutput.java

```diff
@@ -10,4 +10,8 @@ public class GetAllReviewsOutput {
 
     @NonNull
     private final List<Review> reviews;
+
+    public int getSize() {
+        return reviews.size();
```

Comment:
What is the utility of creating a getSize method when the reviews object already has a .size() method available?